### PR TITLE
Improve package and start ARM support

### PR DESCRIPTION
**Pull request by Jibec**

### CODE REVIEW

**Comment by Jibec** [comment]
I don't understand why, but I can't ask @ZeHiro for a review, any idea why?

**Comment by LowMem** [comment]
@Jibec I believe that it is because @ZeHiro is not in @YunoHost-Apps/kresus team because he is not in @YunoHost-Apps organisation…

I just [ask for it there](https://github.com/orgs/YunoHost-Apps/teams/apps-group/discussions/2).

**Comment by Jibec** [comment]
Whoops, I added him yesterday after writing here. But he need to accept I think

**Comment by ZeHiro** [comment]
@Jibec You can now ask for it :) I'll address it today, if I time, and test it this weekend.

**Comment by ZeHiro** [comment]
I agree.

File: scripts/upgrade [hunk 1]
```diff
@@ -162,8 +162,15 @@ ynh_add_systemd_config
 # SECURE FILES AND DIRECTORIES
 #=================================================
 
-# Set right permissions for curl installation
-chown -R $app: $final_path
+# Reset permissions
+chown -R root:root "$final_path"
```
Comment (ZeHiro) [inline scripts/upgrade]: I do not see this in the install script. Did you miss to put it ?

**Comment by ZeHiro** [comment]
I tested on ARM (RPI 2) both with install at root and with a path, and it works very well. Good job.

File: scripts/upgrade (hunk 1 above)
Comment (Jibec) [reply scripts/upgrade]: this isn't needed anymore, we now use a dedicated user for each app:

```
$ grep -iR "chown" ./*
./scripts/restore:chown -R "$app": "$final_path"
./scripts/upgrade:chown -R "$app": "$final_path"
./scripts/install:chown -R "$app": "$final_path"
```

**Pull request merged**